src/clr-angular/forms/select/providers/option-selection.service.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Injectable} from "@angular/core";
+import {Subject} from "rxjs";
+import {Observable} from "rxjs/internal/Observable";
+
+import {ClrOption} from "../option";
+
+@Injectable()
+export class OptionSelectionService {
+    // Array as we need support for selecting multiple options
+    currentSelection: Array<ClrOption> = [];
```

Comment:
This is a public property, but it would be better for the Subject below to emit the current selection instead for consumers to access. 

The only place I see it now is inside of the ClrSelect which can just get the array when the emit happens.

Reply:
I did this because if I remember our conversation on Slack correctly, we decided to follow this pattern rather than the `updateSelection` sending the selection over to the `select`, no? We discussed this I think during the forms PR review recently if I am not wrong.

Reply:
We did talk about it, but the decision was to expose the values through the observable instead of storing the property on the service. See here https://github.com/vmware/clarity/blob/master/src/clr-angular/forms/common/providers/ng-control.service.ts#L20

Reply:
Oh cool. Will update this :-) Thanks @gnomeontherun!